.changeset/curly-mangos-hug.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/admin": patch
+---
+
+Fix: Prevent scrolling of `DialogTitle` and `DialogActions` in `EditDialog`
```

Comment:
```suggestion
Prevent scrolling of `DialogTitle` and `DialogActions` in `EditDialog`
```

It's clear that this is a fix by specifying the patch level for the changeset.

Reply:
012e80cf4c2b8534b9bd3b5437ed526e7c39f0ed